### PR TITLE
DOC: Miscellaneous docstring fixes

### PR DESCRIPTION
**Pull request by jhlegarreta**
- DOC: Fix parameter name in docstring
- DOC: Mark optional parameters as such in docstring
- DOC: Add parameter description to docstring

### CODE REVIEW

File: dipy/workflows/denoise.py [hunk 1]
```diff
@@ -337,7 +337,7 @@ def run(self, input_files, slice_axis=2, n_points=3, num_processes=1,
             use as many cores as possible). 0 raises an error.
         out_dir : string, optional
             Output directory. (default current directory)
-        out_unrig : string, optional
+        out_unring : string, optional
```
Comment (skoudoro) [inline dipy/workflows/denoise.py:340]: there is a typo on the filename also. Can you update this too? thanks

File: dipy/workflows/multi_io.py [hunk 2]
```diff
@@ -134,6 +134,8 @@ def io_iterator(inputs, out_dir, fnames, output_strategy='absolute',
         Controls the behavior of the IOIterator for output paths.
     mix_names : bool
         Whether or not to append a mix of input names at the beginning.
+    out_keys : list, optional
+        Output parameter names.
```
Comment (skoudoro) [inline dipy/workflows/multi_io.py:138]: can you add the optional for `mix_names` and `output_strategy` ?  Thanks

**Review by skoudoro — COMMENTED** [review]
LGTM, can you some additional doc fixes ?

File: dipy/workflows/denoise.py (hunk 1 above)
Comment (jhlegarreta) [reply dipy/workflows/denoise.py:340]: Done in c4d10ed. Push-forced.

File: dipy/workflows/multi_io.py (hunk 2 above)
Comment (jhlegarreta) [reply dipy/workflows/multi_io.py:138]: Done in 7cf0a36. Push-forced.

**Review by skoudoro — APPROVED** [review]
Thanks @jhlegarreta, merging

**Pull request merged**